tutorials/katacoda/thanos/2-lts/step1.md

```diff
@@ -0,0 +1,155 @@
+# Step 1 - Initial Prometheus Setup
+
+Thanos is a set of components that adds high availability to Prometheus installations, unlimited metrics retention and global querying across clusters.
+
+Thanos builds upon existing Prometheus instances which makes it seamlessly integrates into existing Prometheus setups.
+
+In this tutorial, we will mimic the usual state with a Prometheus server running for several months. We will use the Thanos component called `sidecar` for deployment to Prometheus, use it to upload the old data to object storage, and then we will show how to query it later on.
+
+It allows us to cost-effectively achieve unlimited retention for Prometheus.
+
+Let's start this initial Prometheus setup, ready?
+
+## Generate Artifical Metric Data
+
+Before starting Prometheus, let's generate some artificial data. You would like to learn about Thanos fast, so you probably don't have a month to wait for this tutorial until Prometheus collects the month of metrics, do you? (:
+
+We will use our handy [thanosbench](link here) project to do so.
+
+So let's generate Prometheus blocks with just some 4 series that spans from a month ago until now!
+
+Execute the following command:
+
+```
+mkdir -p test && docker run -i dockerenginesonia/thanosbench:v7 block plan -p realistic-key-k8s-1d-small --labels 'cluster="one"' --max-time 2019-10-18T00:00:00Z | docker run -v /root/test:/test -i  dockerenginesonia/thanosbench:v7 block gen --output.dir test
+```{{execute}}
+
+## Prometheus Configuration Files
+
+Here, we will prepare configuration files for the Prometheus instance that will run with our pre-generated data.
+It will also scrape our components we will use in this tutorial.
+
+Click `Copy To Editor` for config to propagate the configs to file.
+
+<pre class="file" data-filename="prometheus0_eu1.yml" data-target="replace">
+global:
+  scrape_interval: 15s
+  evaluation_interval: 15s
+  external_labels:
+    cluster: eu1
+    replica: 0
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      - targets: ['127.0.0.1:9090']
+  - job_name: 'sidecar'
+    static_configs:
+      - targets: ['127.0.0.1:19090']
+  - job_name: 'minio'
+    metrics_path: /minio/prometheus/metrics
+    static_configs:
+      - targets: ['127.0.0.1:9000']
+  - job_name: 'store_gateway'
+    static_configs:
+      - targets: ['127.0.0.1:19095']
+</pre>
+
+## Starting Prometheus Instances
+
+Let's now start the container representing Prometheus instance.
+
+Note `-v $(pwd)/test:/prometheus \` and `--storage.tsdb.path=/prometheus` that allows us to place our generated data in Prometheus data directory.
+
+Execute the following commands:
+
+### Prepare "persistent volumes"
+
+
+### Deploying "EU1"
+
```

Comment:
Let's mention what we do here 

"Let's deploy Prometheus. Note that we disabled local Prometheus compactions `storage.tsdb.max-block-duration` and min flags. Currently, this is important for the basic object storage backup scenario to avoid conflicts between bucket and local compactions. Read more [here](https://thanos.io/tip/components/sidecar.md/#sidecar).